SWIPs/swip-message_to_honey.md

```diff
@@ -26,42 +25,75 @@ This SWIP is part of a series of SWIPs (but can be implemented on it's own). To
+* The `TTL` is set as a variable inside the implementation of the oracle and can be updated.
+* To ensure that a new entry in the 'smarmMessage' object can't become valid before all nodes are updated, the smart-contract or update-policy must ensure that the `validFrom` of new prices must be at least `TTL` seconds in the future.
+* If the oracle cannot be reached when the `TTL` of the last `messagePrices` object is expired, nodes will continue to apply the `prices` as instructed by the expired `messagePrices` object and will start to query the oracle at a more regular interval to re-establish connection. 
 
 ## Rationale
 <!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
-No alternative solutions to updating the *relative* message pricing was considered. It is perhaps worthy to look at how op-code prices (in gas) are updated in Ethereum. Here we also have relative prices and the developers are tweaking the gas-prices on almost every hard-fork (see [opcode re-price discussion](https://ethereum-magicians.org/t/opcode-repricing/3024)). Using an oracle, updatable by a multi-signature of Swarm developers, effectively also allows the developers to set prices (just as in ethereum with ether opcodes). However, the ethereum opcode upgrade mechanism must also be approved by the majority of the hashing power of the Ethereum network (to have a successful hard-fork). This check is missing in the current design for `MsgToHoney` price oracles. 
+No alternative solutions to updating the *relative* message pricing were considered. It is perhaps worthy to look at how op-code prices (in gas) are updated in Ethereum. Here we also have relative prices and the developers are tweaking the gas-prices on almost every hard-fork (see [opcode re-price discussion](https://ethereum-magicians.org/t/opcode-repricing/3024)). Using an oracle, updatable by a governance structure of Swarm developers and stakeholders, effectively also allows the developers to set prices (just as in Ethereum with ether opcodes). However, the Ethereum opcode upgrade mechanism must also be approved by the majority of the hashing power of the Ethereum network (to have a successful hard-fork). This check is missing in the current design for `MsgToHoney` price oracles. 
 
 ## Backwards Compatibility 
 <!--All SWIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The SWIP must explain how the author proposes to deal with these incompatibilities. SWIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
```

Comment:
`before Swarm goes live with incentivization` (appears at the end of this section)

Reply:
Changed!